website/views/__init__.py

```diff
@@ -23,22 +24,78 @@ def account_settings():
     return "To be implemented"
 
 
+paireddevices = [{'text': 'Bathroom Thermostat', 'id': '10', 'type': 'thermostat'},
+                 {'text': 'Kitchen Thermostat', 'id': '20', 'type': 'thermostat'},
+                 {'text': 'Dining Room Thermostat', 'id': '30', 'type': 'thermostat'},
+                 {'text': 'Dining Room Motion Sensor', 'id': '40', 'type': 'motion_sensor'},
+                 {'text': 'Bedroom Motion Sensor', 'id': '50', 'type': 'motion_sensor'},
+                 {'text': 'Bathroom Light Switch', 'id': '60', 'type': 'light_switch'}]
+
+motionactions = ['Turn on', 'Turn Off', 'No Action']
+lightactions = ['Turn Switch on', 'Turn Switch Off', 'No Action']
+thermostatactions = ['Turn on', 'Turn Off', 'No Action', 'Modify Temperature']
+actors = [{'id': '123', 'name': 'Motion Sensor', 'action': motionactions},
+          {'name': 'Light Switch', 'id': '456', 'action': lightactions},
+          {'name': 'Thermostat', 'id': '789', 'action': thermostatactions}]
+motiontriggers = [{'id': '00', 'name': 'When Motion is Detected', 'trigactor': [actors]},
+                  {'id': '01', 'name': 'When No Motion is Detected', 'trigactor': [actors]}]
+thermostattriggers = [{'id': '000', 'name': 'When the temperature is above 22'},
+                      {'id': '1111', 'name': 'When temperature is below 15'}]
+lighttriggers = [
+    {'id': '0000', 'name': 'Lights are on for 4 hours', 'trigactor': [actors], 'trigaction': [lightactions]}]
+
+
 @app.route('/device/actions')
```

Comment:
Could you add the device ID as parameter to this URI, please? e.g.: /device/<string:device_id>